crates/red_knot/src/main.rs

```diff
@@ -9,7 +9,7 @@ use salsa::plumbing::ZalsaDatabase;
 
 use red_knot_server::run_server;
 use red_knot_workspace::db::RootDatabase;
-use red_knot_workspace::site_packages::site_packages_dirs_of_venv;
+use red_knot_workspace::site_packages::VirtualEnvironment;
```

Comment:
It feels a bit surprising that `VirtualEnvironment` is part of the `site_packages` module. Shouldn' we rename the `site_packages` module to `venv`?

Reply:
That might make @carljm upset, since he keeps pointing out that several of the routines in this module really work with any Python installation, not just virtual environments 😆

Reply:
Hmm yeah. Not sure. The hierarchy just feels slightly off to me. But we can also revisit this later.

Reply:
Well, it would depend how you do it :) I was actually going to make a similar comment and decided not to. But I would not want to just rename this module, I would want two modules, one which is specific to venvs and one which contains stuff that works on Python installations in general. But I'm also fine with just keeping it all in one module for now, which is why I didn't make that comment :)

Reply:
Okay, I'll leave this for now; we can come back to it later